scripts/ldap/replication/setupReplicationScript.sh

```diff
@@ -67,71 +70,58 @@ getServerIdFromHostFile()
     id=`expr ${id} + 1`
     done <$host_list
 }
-#Below will get serverid from salt command
-getServerIdWithSalt()
```

Comment:
Have we intentionally removed this?
So we are not taking id from salt command anyways. rit?

Reply:
yes, @shalakadharap  this is intentional, we will not be using salt commands in any way.